pkg/blockstorage/vmware/vmware.go

```diff
@@ -791,6 +791,7 @@ func (ge govmomiError) Format() string {
 	return fmt.Sprintf("[%s]", strings.Join(msgs, "; "))
 }
 
+// nolint:gocognit
```

Comment:
"Directive" type comments shouldn't have a space after the `//`.

```suggestion
//nolint:gocognit
```

Reply:
heh, theres a bunch of places where this needs to be fixed